code/components/openmetrics/openmetrics.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#ifndef OPENMETRICS_H
+#define OPENMETRICS_H
+
+#include <string>
+#include <fstream>
+#include <vector>
+
+#include "ClassFlowDefineTypes.h"
+
+using namespace std;
```

Comment:
I don't know about the rest of the code, but `using namespace` for any namespace in a header is a good candidate for causing problems. Just refer to the types with `std::`.